images/gcb-docker-gcloud/cloudbuild.yaml

```diff
@@ -14,7 +14,7 @@ steps:
     - gcr.io/$PROJECT_ID/gcb-docker-gcloud:latest
 substitutions:
   _GIT_TAG: '12345'
-  _GO_VERSION: 1.20.5
+  _GO_VERSION: 1.20.10
```

Comment:
should we bump to 1.21?

cc @BenTheElder

Reply:
I think it's better to keep the latest `1.20` Version in place as most of the consumers are still on 1.20 version. However please let me know if we need to bump this to 1.21.   @cpanato

Reply:
lets keep that for now

Reply:
maybe we can have both configs? 
cc @BenTheElder @aojea